packages/patrol/lib/src/native/native_automator.dart

```diff
@@ -73,6 +74,22 @@ class NativeAutomatorConfig {
   /// Time to wait for native views to appear.
   final Duration findTimeout;
 
+  /// Whether to show the keyboard when entering text.
+  ///
+  /// The default is to always show the keyboard when entering text, and only
+  /// dismiss it after tapping on another UI element. However, that "another UI
```

Comment:
> after tapping on another UI element

is this Patrol's behavior or does Patrol expect this behavior from the app?

If it's the latter, please describe it more specifically.

Reply:
This depends on the app, screen dimensions of the device you're running on (e.g. on iOS Simulator you can completely disable the keyboard), basically everything. Every case is different - even the exact same test run a few times can result in different behavior.